source/promap.js

```diff
@@ -0,0 +1,13 @@
+import _curry3 from './internal/_curry3';
+import pipe from './pipe';
+
+
+var promap = _curry3(function promap(f, g, profunctor) {
+  return typeof profunctor['fantasy-land/promap'] === 'function'
+    ? profunctor['fantasy-land/promap'](f, g)
+    : typeof profunctor.promap === 'function'
+      ? profunctor.promap(f, g)
```

Comment:
I don't think we should dispatch to `promap`.

Reply:
I noticed some other methods dispatch similarly, so included it just in case. I have no opinion in this regard so fine with excluding it

Reply:
@davidchambers: Why not?  That is still so far how Ramda functions tend to work.  Is there something about `promap` that is different, or is this a more general objection?

However, I would rather use the mechanism we have in place, using `_dispatchable`.  See, for example [`chain`][ch].


  [ch]: https://github.com/ramda/ramda/blob/master/source/chain.js

Reply:
> That is still so far how Ramda functions tend to work.

I thought this was due to the way Fantasy Land worked four years ago.